macros/parserGraphTool.pl

```diff
@@ -228,6 +235,17 @@ sub new {
 }
 
 our %graphObjectTikz = (
+	point => {
+		code => sub {
+			my $self = shift;
+			my ($x, $y) = @{$_->{data}[2]{data}};
```

Comment:
This should be 
```suggestion
			my ($x, $y) = @{$_->{data}[1]{data}};
```
0 is the string "point", and 1 is the point itself.  2 is undefined.